packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -70,6 +70,12 @@ enum TimePickerEntryMode {
 
   /// Text input.
   input,
+
+  /// Only tapping/dragging on a clock dial.
+  dialOnly,
+
+  /// Only text input.
+  inputOnly
```

Comment:
Can we expand on the docs for all four of of these values to be consistent with how they are described for the equivalent in the Date Picker: 

https://github.com/flutter/flutter/blob/cb9a1d623ccd460c3744ed019af76496c47a353e/packages/flutter/lib/src/material/date.dart#L158

Reply:
Hello @darrenaustin,
Thank you for the feedback. I made the change in the TimePickerEntryMode doc to match the DatePicker equivalent.
Benjamin.